static/_redirects

```diff
@@ -1,5 +1,137 @@
-# Redirect attempts to get to docs to a more helpful location.
-/v1/gonum/*	https://pkg.go.dev/gonum.org/v1/gonum/	302
-/v1/plot/*	https://pkg.go.dev/gonum.org/v1/plot/	302
-/v1/netlib/*	https://pkg.go.dev/gonum.org/v1/netlib/	302
-/v1/exp/*	https://pkg.go.dev/gonum.org/v1/exp/	302
\ No newline at end of file
+# Generated by generate_directs.sh; DO NOT EDIT.
+# Redirects to get to docs to a more helpful location.
+/v1/gonum/graph/formats/dot/internal/parser/* https://pkg.go.dev/gonum.org/v1/gonum/graph/formats/dot/internal/parser/ 302
```

Comment:
couldn't we sort those? (mainly to have a stable output)
probably injecting a `| sort` would do.

Reply:
ah... no. well not *as simple* as just adding a dumb `|sort`...

Reply:
They are sorted, by length of line. The redirect matcher works first found, so this makes choices specific.

Reply:
Making the sort, `sort -k1,1nr -k2` would make it completely stable.

Reply:
Done